### ISSUE TITLE
`media-feature-range-notation` false positive

### ISSUE THREAD

**Issue by FloEdelmann** [stylelint/stylelint#6816]
### What steps are needed to reproduce the bug?

```scss
@media screen and (min-height: 26rem) {}
```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
     "media-feature-range-notation": "context"
  }
}
```

### How did you run Stylelint?

`npx stylelint style.css`

### Which version of Stylelint are you using?

15.6.0

### What did you expect to happen?

No errors.

### What actually happened?

The following problem was reported:

```
Expected "context" media feature range notation (media-feature-range-notation)
```

### Does the bug relate to non-standard syntax?

No, only native CSS syntax is involved.

**Comment by ybiquitous** [comment]
@FloEdelmann Thanks for the report. I can confirm the error with [this demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBbGUCWBDABAZzACcYYA7HLUqHAChQ1IFoALGDAc2YBd4cAmAGzEUAShzAAviAA0IAGYYANjAByWNAhAwAHuoAOygHRg8eGeAikF7TcAA65HHZCEArsrzPe9xzicg0TCxGORgsLldiRkJKdhhGUggucIxLL39IUi4dLmcHPwkHKVlM6wAxCEIUcM0AKzxLc1g9M0QfP2c8LgBPZUUGXIR-RXCYLudpfP8u3ph+rMZSjkYuyigsQih05xHs8ZAikAkgA).

However, I don't understand why `(height >= 26rem)` is a problem. Could you please explain it?

**Comment by FloEdelmann** [comment]
Oh, sorry, that indeed seems to be correct. I suspected it to be auto-fixed to this instead:

```
@media (min-height: 26rem) {}
```

… which would obviously be wrong. However, the actual autofix is correct:

```
@media screen and (height >= 26rem) {}
```

So I should have checked it better before opening an issue.
